contracts/bridge/Bridge.sol

```diff
@@ -46,37 +46,42 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
 
     /**
      * Event to log the withdrawal of a token from the Bridge.
```

Comment:
Um... I think RequestValueTransfer is for depositing. 
If my thinking is correct, how about edit comment here 'withdrawal' and L49?